library/src/main/java/org/apache/apex/malhar/lib/state/spillable/SpillableByteArrayListMultimapImpl.java

```diff
@@ -90,7 +100,7 @@ public SpillableByteArrayListMultimapImpl(SpillableStateStore store, byte[] iden
     this.bucket = bucket;
     this.serdeKey = Preconditions.checkNotNull(serdeKey);
```

Comment:
Later on the class of serde is used to determine how to deal with serialization. That kind of breaks encapsulation and isn't obvious when supplying the serde and should be documented.